src/lib/EventSubscriber/AppRendererSubscriber.php

```diff
@@ -51,7 +51,7 @@ public function renderApp(FilterResponseEvent $event)
 
         $response = $event->getResponse();
 
-        if ($response->isRedirect()) {
+        if ($response->isRedirect() || $response instanceof NoRenderResponse) {
```

Comment:
Or you could implement `isRedirect` in `ResetResponse`. It has a `Location` header (or close enough), I think that's clean.